test/datasource/maven.spec.js

```diff
@@ -86,7 +86,7 @@ describe('datasource/maven', () => {
     it('should return versions in all repositories for a specific library', async () => {
       const releases = await datasource.getPkgReleases({
         ...config,
-        lookupName: 'mysql/mysql-connector-java',
+        lookupName: 'mysql:mysql-connector-java',
```

Comment:
Not sure should I use it here too, but unit test are passing as well as for real repo
@rarkins